check_source.py

```diff
@@ -152,10 +153,21 @@ def check_source_submission(self, source_project, source_package, source_revisio
             if devel_project:
                 if ((source_project != devel_project or source_package != devel_package) and
                    not (source_project == target_project and source_package == target_package)):
-                    # Not from proper devel project/package and not self-submission.
-                    self.review_messages['declined'] = 'Expected submission from devel package %s/%s' % (
-                        devel_project, devel_package)
-                    return False
+                    # check if the devel project & package are using scmsync
+                    # => waive the devel project source submission requirement
+                    meta = ET.fromstringlist(show_package_meta(self.apiurl, devel_project, devel_package))
+                    scm_sync = meta.find('scmsync')
+                    if (
+                            not scm_sync or
+                            (
+                                scm_sync and
+                                not scm_sync.text.startswith(f"https://src.opensuse.org/pool/{source_package}")
+                            ) or source_project not in self.allowed_scm_submission_sources
```

Comment:
If you want create subprojects per SR, this needs to be a "prefix" check.

Reply:
Should be done now, but please double check it.